src/care/policies.h

```diff
@@ -69,9 +69,13 @@ using RAJADeviceExec = RAJA::seq_exec;
 
 //  reduction kernel policy
 #if defined(__HIPCC__)
-using RAJAReductionExec = RAJA::hip_exec_occ_calc<CARE_CUDA_BLOCK_SIZE, CARE_CUDA_ASYNC>;
+using RAJAReductionExec = RAJA::hip_exec_with_reduce<256, CARE_CUDA_ASYNC>;
+#elif defined(__CUDACC__)
+using RAJAReductionExec = RAJA::cuda_exec_with_reduce<256, CARE_CUDA_ASYNC>;
+#elif defined(_OPENMP) && defined(RAJA_ENABLE_OPENMP) // CARE_GPUCC
+using RAJAReductionExec = RAJA::omp_parallel_for_exec
 #else
-using RAJAReductionExec = RAJADeviceExec;
+using RAJAReductionExec = RAJADeviceExec
```

Comment:
should we use consistent semicolons for these using statements?

Reply:
yes. Fixed.